testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/HydraIdentityProvider.java

```diff
@@ -88,6 +93,19 @@ public void extendEnvironment(Environment.Builder builder)
         builder.containerDependsOn(hydra.getLogicalName(), hydraConsent.getLogicalName());
         builder.containerDependsOn(hydra.getLogicalName(), migrationContainer.getLogicalName());
         builder.containerDependsOn(hydra.getLogicalName(), databaseContainer.getLogicalName());
+
+        builder.configureContainers(dockerContainer -> {
+            if (isPrestoContainer(dockerContainer.getLogicalName())) {
+                dockerContainer
+                        .withCopyFileToContainer(
+                                forHostPath(configDir.getPath("cert")),
+                                CONTAINER_PRESTO_ETC + "/hydra/cert");
+            }
+        });
+
+        builder.configureContainer(TESTS, dockerContainer -> dockerContainer.withCopyFileToContainer(
+                forHostPath(dockerFiles.getDockerFilesHostPath("conf/tempto/tempto-configuration-for-docker-oauth2.yaml")),
```

Comment:
Wouldn't it be easier to just more `tempto-configuration-for-docker-oauth2.yaml` to `common/hydra-identity-provider`?